thrift/src/test/java/com/linecorp/armeria/server/thrift/ThriftServiceTest.java

```diff
@@ -636,7 +636,8 @@ private static void invoke0(THttpService service, HttpData content,
                 ServiceRequestContextBuilder.of(req)
                                             .eventLoop(eventLoop.get())
                                             .serverConfigurator(builder -> {
-                                                builder.blockingTaskExecutor(ImmediateEventExecutor.INSTANCE);
+                                                builder.blockingTaskExecutor(ImmediateEventExecutor.INSTANCE,
+                                                                             true);
```

Comment:
`false` is probably better, although it will not make much difference. `ImmediateEventExecutor` never terminates.